workbench-core/environments/src/services/environmentTypeConfigService.test.ts

```diff
@@ -15,53 +39,57 @@ import {
 } from '@aws-sdk/client-dynamodb';
 import { marshall } from '@aws-sdk/util-dynamodb';
 import { resourceTypeToKey } from '@aws/workbench-core-base';
+import DynamoDBService from '@aws/workbench-core-base/lib/aws/helpers/dynamoDB/dynamoDBService';
 import { mockClient } from 'aws-sdk-client-mock';
+
 import EnvironmentTypeConfigService from './environmentTypeConfigService';
+import EnvironmentTypeService from './environmentTypeService';
 
 describe('environmentTypeConfigService', () => {
-  beforeAll(() => {
-    process.env.AWS_REGION = 'us-east-1';
-  });
-  const ddbMock = mockClient(DynamoDBClient);
-  beforeEach(() => {
-    jest.clearAllMocks();
-    ddbMock.reset();
-  });
   const TABLE_NAME = 'exampleDDBTable';
-  const envTypeConfigService = new EnvironmentTypeConfigService({ TABLE_NAME });
-  const envTypeId = '1b0502f3-121f-4d63-b03a-44dc756e4c20';
+  const ddbServiceMock = new DynamoDBService({ region: 'us-east-1', table: TABLE_NAME });
+  const envTypeMock = new EnvironmentTypeService({ TABLE_NAME });
+  const envTypeConfigService = new EnvironmentTypeConfigService(envTypeMock, ddbServiceMock);
+  const ddbMock = mockClient(DynamoDBClient);
+
   const envTypeConfig = {
     createdAt: '2022-06-17T16:28:40.360Z',
-    updatedBy: 'owner-123',
-    createdBy: 'owner-123',
     name: 'config 1',
-    allowRoleIds: [],
-    resourceType: 'envTypeConfig',
+    //
     // provisioningArtifactId: 'pa-dewjn123',
     params: [],
     updatedAt: '2022-06-17T21:25:24.333Z',
-    sk: `${resourceTypeToKey.envType}#${envTypeId}${resourceTypeToKey.envTypeConfig}#${envTypeConfigId}`,
-    owner: 'owner-123',
+    //sk: `${resourceTypeToKey.envType}#${envTypeId}${resourceTypeToKey.envTypeConfig}#${envTypeConfigId}`,
     description: 'Example config 1',
     id: envTypeConfigId,
-    pk: 'ETC',
+    //pk: `${resourceTypeToKey.envType}#${envTypeId}${resourceTypeToKey.envTypeConfig}#${envTypeConfigId}`,
```

Comment:
Intentional?

Reply:
forgot to delete commented line, we don't really need the pk, sk as the object returned does not contain them

Reply:
comments deleted